lib/presentation/setting/setting_page.dart

```diff
@@ -0,0 +1,129 @@
+        ),
+        title: const Text(
+          '설정',
+          style: PicpleTypography.title1,
+        ),
+        centerTitle: true,
+      ),
+      body: Container(
+        color: PicpleColors.white,
+        width: double.infinity,
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            _SettingItem(
+              icon: Icons.person_outline,
+              label: '프로필 수정',
+              onTap: () {},
+            ),
+            _SettingItem(
+              icon: Icons.description_outlined,
+              label: '서비스 이용약관',
+              onTap: () {},
+            ),
+            _SettingItem(
+              icon: Icons.location_on_outlined,
+              label: '위치정보 이용약관',
+              onTap: () {},
+            ),
+            _SettingItem(
+              icon: Icons.assignment_outlined,
+              label: '개인정보 처리방침',
+              onTap: () {},
+            ),
+            _SettingItem(
+              icon: Icons.logout,
+              label: '로그아웃',
+              onTap: () {}
+            ),
+            const Padding(
+              padding: EdgeInsets.symmetric(horizontal: 24, vertical: 16),
+              child: Row(
+                children: [
+                  Icon(Icons.info_outline, color: PicpleColors.gray5, size: 24),
+                  SizedBox(width: 16),
+                  Text(
+                    '버전 정보',
+                    style: PicpleTypography.body1,
+                  ),
+                  Spacer(),
+                  Text(
+                    'v1.0.0',
+                    style: PicpleTypography.body1,
+                  ),
+                ],
+              ),
+            ),
+            Center(
+              child: TextButton(
+                onPressed: () {},
+                child: const Text(
+                  '탈퇴하기',
+                  style: TextStyle(
+                    color: PicpleColors.gray5,
+                    fontSize: 14,
+                    decoration: TextDecoration.underline,
+                  ),
+                ),
+              ),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+class _SettingItem extends StatelessWidget {
+  final IconData icon;
+  final String label;
+  final VoidCallback? onTap;
+
+  const _SettingItem({
+    required this.icon,
+    required this.label,
+    this.onTap,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return InkWell(
+      onTap: onTap,
+      child: Padding(
+        padding: const EdgeInsets.symmetric(horizontal: 24, vertical: 16),
+        child: Row(
+          children: [
+            Icon(icon, color: PicpleColors.gray5, size: 24),
+            const SizedBox(width: 16),
+            Text(
+              label,
+              style: PicpleTypography.body1,
+            ),
+            const Spacer(),
+            const Icon(Icons.chevron_right, color: PicpleColors.gray5),
+          ],
+        ),
+      ),
+    );
```

Comment:
```suggestion
@override
Widget build(BuildContext context) {
  return Material(
    child: InkWell(
      onTap: onTap,
      child: Padding(
        padding: const EdgeInsets.symmetric(horizontal: 24, vertical: 14),
        child: Row(
          children: [
            Icon(icon, color: PicpleColors.gray5, size: 24),
            const SizedBox(width: 16),
            Text(
              label,
              style: PicpleTypography.body1.copyWith(
                color: PicpleColors.black,
              ),
            ),
            const Spacer(),
            const Icon(Icons.chevron_right, color: PicpleColors.gray5),
          ],
        ),
      ),
    ),
  );
```

C : Material이 상위 위젯이여야 RippleEffect 생겨요
패딩 조절 및 글꼴 색상 조정